packages/devextreme/js/__internal/ui/map/m_map.ts

```diff
@@ -82,6 +86,13 @@ const Map = Widget.inherit({
     ]);
   },
 
+  _setDeprecatedOptions() {
+    this.callBase();
+    extend(this._deprecatedOptions, {
+      'googleMapConfig.useAdvancedMarkers': { since: '24.2', message: 'Google deprecated the original map markers. Transition to advanced markers for future compatibility.' },
```

Comment:
@vladaskorohodova  check deprecation message please

Reply:
I've alredy discussed this with Vlada